src/java/org/apache/cassandra/db/DeletionTime.java

```diff
@@ -195,42 +195,100 @@ public static Serializer getSerializer(Version version)
             return legacySerializer;
     }
 
-    // Serializer for Usigned Integer ldt
+    /* Serializer for Usigned Integer ldt
+     *
+     * ldt is encoded as a uint in seconds since unix epoch, it can go up o 2106-02-07T06:28:13+00:00 only. 
+     * Since mfda is micros since the Unix Epoch with 7 bytes we can encode 2^56 ~= 1142 years. That leaves 
+     * 1 free byte we can use to store flags for sentinel values with some space saving.
+     *
+     * Currently only a flag for the LIVE ldt is being used.
+     */
     public static class Serializer implements ISerializer<DeletionTime>
     {
+        private final static int IS_LIVE_DELETION = 0x01;
+        // Long.MAX_VALUE sentinel value equivalent in 7bytes
+        public final static long MAX_MFDA = 0xffffffffffffffL;
```

Comment:
wouldn't `(1 << 56) - 1` be simpler and easier to read? wdyt?
or
(1 << (7 * 8)) - 1

Reply:
Yep yep yep. I like `(1L << (7 * 8)) - 1` better

Reply:
Fixed in the latest commit already. Resolving but feel free to reopen if needed.